ulib/fs/FStar_Int16.fs

```diff
@@ -0,0 +1,85 @@
+module FStar_Int16
+(* !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! *)
+(* THIS FILE IS BASED ON AUTOGENERATED ml/FStar_Int16.ml FILE! *)
```

Comment:
This is not currently autogenerated, is it? I could tweak the scripts to make that work.

Reply:
Yeah, it's not generated currently. I think it's a good idea as it will make it easier to keep them in sync with ml versions.